kubetest/kops.go

```diff
@@ -516,3 +588,52 @@ func (k kops) Publish() error {
 		return gcsWrite(k.kopsPublish, []byte(k.kopsVersion))
 	})
 }
+
+// getRandomAWSZones looks up all regions, and the availability zones for those regions.  A random
+// region is then chosen and the AZ's for that region is returned.  If HA is required a regions with 3
+// or more zones is returned.
```

Comment:
Maybe replace "  If HA is required a regions with 3 or more zones is returned." with "At least masterCount zones will be returned, all in the same region" or something like that?